kernel/src/process/posix_thread/mod.rs

```diff
@@ -53,6 +53,9 @@ pub struct PosixThread {
     /// Process credentials. At the kernel level, credentials are a per-thread attribute.
     credentials: Credentials,
 
+    /// The file system information of the thread.
+    fs: RwMutex<Arc<ThreadFsInfo>>,
+
```

Comment:
Since the `fs` field is added in the `PosixThread`, it should also be unshred in `unshare_fs` (See kernel/src/process/namespace/unshare.rs)